### PR TITLE
[FIX] this week's ci errors

### PR DESCRIPTION
**Pull request by hbrunn**
This is meant to fix the problems runbot sees with the intrastat branch currently

### CODE REVIEW

File: intrastat_base/models/product_template.py [hunk 1]
```diff
@@ -33,13 +33,14 @@ class ProductTemplate(models.Model):
...
-                    "'%s'"
-                    % (self.name, self.type)))
+        for this in self:
+            if this.is_accessory_cost and this.type != 'service':
+                raise ValidationError(
+                    _("The option 'Is accessory cost?' should only be "
+                        "activated on 'Service' products. You have activated "
+                        "this option for the product '%s' which is of type "
+                        "'%s'") %
+                    (self.name, self.type))
```
Comment (alexis-via) [inline intrastat_base/models/product_template.py]: self -> this

File: intrastat_product/models/intrastat_product_declaration.py [hunk 2]
```diff
@@ -197,29 +197,32 @@ def _check_year(self):
...
+    @api.multi
     @api.depends('company_id')
     def _compute_company_country_code(self):
-        if self.company_id:
-            self.company_country_code = \
-                self.company_id.country_id.code.lower()
+        for this in self:
+            if this.company_id:
+                this.company_country_code = \
+                    self.company_id.country_id.code.lower()
```
Comment (alexis-via) [inline intrastat_product/models/intrastat_product_declaration.py]: self -> this

File: intrastat_product/models/intrastat_product_declaration.py [hunk 3]
```diff
@@ -812,11 +815,12 @@ class IntrastatProductComputationLine(models.Model):
...
         help="Country of origin of the product i.e. product 'made in ____'")
 
-    @api.one
+    @api.multi
     @api.depends('transport_id')
     def _check_validity(self):
         """ TO DO: logic based upon fields """
-        self.valid = True
+        for this in self:
+            this.valid = True
```
Comment (alexis-via) [inline intrastat_product/models/intrastat_product_declaration.py:823]: We can avoid the loop, no ?

File: intrastat_base/models/product_template.py (hunk 1 above)
Comment (hbrunn) [inline intrastat_base/models/product_template.py]: ooops, done

File: intrastat_product/models/intrastat_product_declaration.py (hunk 3 above)
Comment (hbrunn) [inline intrastat_product/models/intrastat_product_declaration.py:823]: no, compute functions can be called on any amount of records simultaneously

**Comment by hbrunn** [comment]
what shameful misses, thanks for looking closely!

**Comment by hbrunn** [comment]
@alexis-via it there still anything to do before you can merge?

**Comment by alexis-via** [comment]
thanks for your work !

**Pull request merged**